mzLib/MzML/Mzml.cs

```diff
@@ -373,16 +377,24 @@ private static MsDataScan GetMsDataOneBasedScanFromConnection(Generated.mzMLType
                     intensityArray |= cv.accession.Equals(_intensityArray);
                 }
 
-                double[] data = ConvertBase64ToDoubles(binaryData.binary, compressed, is32bit);
-                if (mzArray)
+                try
```

Comment:
Fair enough! Could you comment the `try` and `catch` blocks with their intended purposes?

Reply:
yes. of course